src/bun.js/webcore/streams.zig

```diff
@@ -52,6 +52,59 @@ pub const ReadableStream = struct {
     value: JSValue,
     ptr: Source,
 
+    pub const Strong = struct {
+        held: JSC.Strong = .{},
+        self: ?ReadableStream = null,
```

Comment:
JSC.Strong has the inner value here, it's not a good idea to have both fields

Reply:
fixed in https://github.com/oven-sh/bun/pull/4418/commits/28eafc84e0d67e910bd9af656426aa3cdfba5c12 and https://github.com/oven-sh/bun/pull/4418/commits/e4ae61ac86219e56a766cd85712bd3d1749f7f4c